### ISSUE TITLE
[Question] Is it possible to use kuttl with minikube

### ISSUE THREAD

**Issue by milindw96** [kudobuilder/kuttl#304]
Hi All,
I am trying to use kuttl with minikube but I don't find any doc related to it, my base machine is centos 8 and 
```minikube version: v1.21.0``` 

Thanks in Advance :-)

**Comment by kensipe** [comment]
absolutely...   kuttl will work against any k8s cluster (or is designed to be).   I was using minikube a lot in the beginning and mainly use kind now.  Configuring kind is built into kuttl for convenience... but we have lots of folks using against a variety of clusters.

If you have KUBECONFIG set to minikube... kuttl will run with it as it's test cluster.

**Comment by milindw96** [comment]
If possible can u share some doc related to it.. will try it

**Comment by kensipe** [comment]
Sorry for delay (holidays here).   I certainly will!

**Comment by milindw96** [comment]
any Update ?
